src/qt/createwalletdialog.cpp

```diff
@@ -60,3 +60,8 @@ bool CreateWalletDialog::isMakeBlankWalletChecked() const
 {
     return ui->blank_wallet_checkbox->isChecked();
 }
+
+bool CreateWalletDialog::isDescriptorWalletChecked() const
```

Comment:
f5204ed3fe1b2a059e35e54248d38eac88c629e0 could be split between GUI and RPC.

Reply:
Meh, it's not really a big difference.

Reply:
Yeah, just keep in in mind if something else needs to be changed there, or if - god forbid - we get bike shedding over the new check box.